doc/api/string_decoder.md

```diff
@@ -71,6 +71,7 @@ substitution characters appropriate for the character encoding.
 
 If the `buffer` argument is provided, one final call to `stringDecoder.write()`
 is performed before returning the remaining input.
+After `end` is called, the `stringDecoder` object can be reused for new input.
```

Comment:
```suggestion
After `end()` is called, the `stringDecoder` object can be reused for new input.
```

Reply:
@Trott - thanks for the review, I have followed your suggestion. Please have a look.